server/methods/setUserActiveStatus.js

```diff
@@ -21,8 +21,35 @@ Meteor.methods({
 			});
 		}
 
-		setUserActiveStatus(userId, active, confirmRelenquish);
+		const user = Meteor.users.findOne({
+			_id: userId,
+		}, {
+			fields: {
+				_id: 1,
+				roles: 1,
+				active: 1,
+			},
+		});
 
+		// Check if user is the last ACTIVE admin and Prevent removing last user from admin role
+		const userIsAdmin = user.roles.indexOf('admin') > -1;
```

Comment:
You can do this right in the query 😬 and then check if the query returns something. If there's no return for a user with the ID and the `admin` role, then the user is not an admin